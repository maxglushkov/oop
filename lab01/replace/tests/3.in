0123123123456789
