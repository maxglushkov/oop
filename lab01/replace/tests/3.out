0123АбвАбвГ56789
